Makefile

```diff
@@ -144,7 +152,7 @@ test-plugins:
 	pip install haas
 	haas simphony -v
 	haas jyulb -v
-	haas simlammps -v
+	LD_LIBRARY_PATH=./lib/:$LD_LIBRARY_PATH haas simlammps -v
```

Comment:
hmm... if simlammps now requires dynamic libraries then these could be copied to the virtual-env `lib` directory (similar to what we do for copying the executables to the `bin` directory. I think that should work